### DISCUSSION TITLE
hope   add Drawer.Footer to Drawer

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#4540] [Feature requests]
This is a very common layout in the project, hope to be native support
The current components are too simple

**Comment by chrisbendel** [comment]
Seconding this!

I have been struggling to find a clear way to have some fixed content at the bottom of a drawer, with an indeterminate amount of content (which could be scrollable) in the main `<Drawer.Body>`

Maybe there is an example out there someone has gotten working @rtivital ? 

Playing around with the height of the Body (ie `h='100%'`) is taking the `<Drawer.Header` into account and the scrollbar is going further than needed 🤔

**Comment by Pyakz** [comment]
up

**Comment by erwinstone** [comment]
I need this too. For a temporary solution, I use:

```jsx
      <Drawer.Root opened={opened} onClose={close}>
        <Drawer.Overlay />
        <Drawer.Content>
          <Drawer.Header>
            <Drawer.Title>Drawer title</Drawer.Title>
            <Drawer.CloseButton />
          </Drawer.Header>
          <Drawer.Body>Drawer content</Drawer.Body>
          <Drawer.Header bottom={0}>Footer</Drawer.Header>
        </Drawer.Content>
      </Drawer.Root>
```

**Comment by saadpocalypse** [comment]
Just opened a [PR](https://github.com/mantinedev/mantine/pull/9077) that would resolve this issue.
